### ISSUE TITLE
Unable to create a cert with Ledger

### ISSUE THREAD

**Issue by Galadrin** [akash-network/support#22]
CLI and Keplr both return `Error: Unexpected characters`  when creating a client certificat.

```
~/go/bin/akash tx cert generate client --from ledger --node https://rpc.cros-nest.com:443/akash
Default sign-mode 'direct' not supported by Ledger, using sign-mode 'amino-json'.
Error: Unexpected characters
```

**Comment by Galadrin** [comment]
* akash version :
 ```
version: 0.20.0
commit: 9c376e978213fba72e1023b829d780f1f4ce64e5
build_tags: '"osusergo,netgo,ledger,static_build"'
go: go version go1.19.4 linux/amd64
```
 * ledger firmware version
 Ledger Nano X version 2.0.2
 * version of the cosmos app on the ledger
 version v2.34.9
 * host os type
Linux mint 21.1 Vera

**Comment by troian** [comment]
fix will be released within `v0.24.0`
